diff --git a/node_modules/@jackallabs/jackal.js/dist/index.esm.js b/node_modules/@jackallabs/jackal.js/dist/index.esm.js
index 3a42e17..d1073ed 100644
--- a/node_modules/@jackallabs/jackal.js/dist/index.esm.js
+++ b/node_modules/@jackallabs/jackal.js/dist/index.esm.js
@@ -4,7 +4,7 @@ var __publicField3 = (obj, key, value) => __defNormalProp3(obj, typeof key !== "
 import { defaultRegistryTypes, AminoTypes, createProtobufRpcClient, StargateClient, SigningStargateClient, QueryClient } from "@cosmjs/stargate";
 import { connectComet } from "@cosmjs/tendermint-rpc";
 import { Registry, DirectSecp256k1HdWallet } from "@cosmjs/proto-signing";
-import { Responses } from "@cosmjs/tendermint-rpc/build/tendermint34/adaptor";
+import { Responses } from "@cosmjs/tendermint-rpc/build/tendermint34/adaptor/index.js";
 import { Responses as Responses$1 } from "@cosmjs/tendermint-rpc/build/comet38/adaptor";
 import { assertDefined } from "@cosmjs/utils";
 import { createVestingAminoConverters, createFeegrantAminoConverters, createIbcAminoConverters, createStakingAminoConverters, createGovAminoConverters, createDistributionAminoConverters, createBankAminoConverters, createAuthzAminoConverters } from "@cosmjs/stargate/build/modules";
